<ui version="4.0" >
 <class>SearchersList</class>
 <widget class="QWidget" name="SearchersList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <widget class="QTreeView" name="SearchersView_" />
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QPushButton" name="ButtonAdd_" >
          <property name="text" >
           <string>Add...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonRemove_" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>ButtonRemove_</zorder>
       <zorder>ButtonAdd_</zorder>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Info</string>
       </property>
       <layout class="QFormLayout" name="formLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
